packages/storybook/package.json

```diff
@@ -49,11 +49,11 @@
     "@govuk-react/unordered-list": "^0.9.0",
     "@govuk-react/visually-hidden": "^0.9.0",
     "@govuk-react/warning-text": "^0.9.0",
-    "@storybook/addon-actions": "6.2.9",
-    "@storybook/addon-docs": "6.2.9",
-    "@storybook/addon-knobs": "6.2.9",
-    "@storybook/addon-links": "6.3.2",
-    "@storybook/addons": "6.2.9",
+    "@storybook/addon-actions": "6.3.5",
+    "@storybook/addon-docs": "6.3.5",
+    "@storybook/addon-knobs": "6.3.0",
+    "@storybook/addon-links": "6.3.5",
+    "@storybook/addons": "6.3.5",
     "@storybook/react": "6.3.5",
```

Comment:
hmm we're on 6.3.5 already are we? 🤔 

I take it you've searched for "6.2.9" and updated everywhere?

Why is addon knobs only updated to 6.3.0?

Reply:
I tried to update addon-knobs too, but found it's been deprecated and 6.3.0 is the latest version - replacement is addon-controls

Reply:
oh right